app/controllers/api/v1/credit_cards_controller.rb

```diff
@@ -0,0 +1,11 @@
+class Api::V1::CreditCardsController < Api::V1::BaseController
+
+  private
+
+  def credit_card_params
+    params.require(:credit_card).permit(:first_name, :last_name, :number, :month, :year, :last_four)
+  end
+end
+
+
+
```

Comment:
remove unnecessary whitespace. with further denote this with a comment that just reads "WS"